fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/metadata/KeyExpressionTest.java

```diff
@@ -766,6 +768,21 @@ void testSplitConcat() {
                 evaluate(splitConcat, numbers));
     }
 
+    @Test
+    void testCanProperlyGetValueExpressionFromKeyWithValue() {
+        /*
+         * Tests for a degenerate case of KeyWithValueExpressions
+         */
+        List<KeyExpression> fields = new ArrayList<>();
+        fields.add(recordType());
+        fields.add(Key.Expressions.field("col0"));
+        fields.add(Key.Expressions.field("col1"));
+        KeyWithValueExpression kwve = new KeyWithValueExpression(Key.Expressions.concat(fields),2);
```

Comment:
```suggestion
        KeyWithValueExpression kwve = new KeyWithValueExpression(Key.Expressions.concat(fields), 2);
```

It looks like there are missing spaces on some other lines as well